### PR TITLE
moved copyProjectDataToPrmTiming to look on prm and not tmp

### PR DESCRIPTION
**Pull request by RoanKanninga**

### CODE REVIEW

**Review by kdelange — APPROVED** [review]

File: bin/pipelineTiming.sh [hunk 1]
```diff
@@ -310,22 +310,22 @@ do
...
 	## step 3, check the copyProjectDataToPrm
-	if [ -e "${logsDir}/${project}/${run}.copyProjectDataToPrm.finished" ]
+	if ssh "${HOSTNAME_PRM}" test -e "/groups/${GROUP}/${PRM_LFS}/logs/${project}/${run}.copyProjectDataToPrm.finished"
 	then
 		echo -e "copyProjectDataToPrm is finished for project ${project}" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
 		touch "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrmTiming.finished"
 		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "copyProjectDataToPrm finished for project ${project}"
 	else
-		timeStampCopyProjectDataToPrm=$(find "${logsDir}/${project}/" -type f -mmin +240 -iname "${run}.copyProjectDataToPrm.started")
+		timeStampCopyProjectDataToPrm=$(ssh "${HOSTNAME_PRM}" "find "/groups/${GROUP}/${PRM_LFS}/logs/${project}/" -type f -mmin +240 -iname "${run}.copyProjectDataToPrm.started""
```
Comment (pneerincx) [inline bin/pipelineTiming.sh:319]: Missing closing `)` at the end of the line.

Comment (pneerincx) [reply bin/pipelineTiming.sh:319]: Has this been tested? Script cannot run as is....

**Review by kdelange — APPROVED** [review]

File: bin/pipelineTiming.sh [hunk 2]
```diff
@@ -310,22 +310,22 @@ do
...
 	## step 3, check the copyProjectDataToPrm
-	if [ -e "${logsDir}/${project}/${run}.copyProjectDataToPrm.finished" ]
+	if ssh "${HOSTNAME_PRM}" test -e "/groups/${GROUP}/${PRM_LFS}/logs/${project}/${run}.copyProjectDataToPrm.finished"
 	then
 		echo -e "copyProjectDataToPrm is finished for project ${project}" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
 		touch "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrmTiming.finished"
 		log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "copyProjectDataToPrm finished for project ${project}"
 	else
-		timeStampCopyProjectDataToPrm=$(find "${logsDir}/${project}/" -type f -mmin +240 -iname "${run}.copyProjectDataToPrm.started")
+		timeStampCopyProjectDataToPrm=$(ssh "${HOSTNAME_PRM}" "find "/groups/${GROUP}/${PRM_LFS}/logs/${project}/" -type f -mmin +240 -iname "${run}.copyProjectDataToPrm.started"")
```
Comment (pneerincx) [inline bin/pipelineTiming.sh:319]:
```suggestion
		timeStampCopyProjectDataToPrm=$(ssh "${HOSTNAME_PRM}" "find \"/groups/${GROUP}/${PRM_LFS}/logs/${project}/\" -type f -mmin +240 -iname \"${run}.copyProjectDataToPrm.started\"")
```

File: bin/pipelineTiming.sh [hunk 3]
```diff
@@ -310,22 +310,22 @@ do
...
 		then
-			echo -e "copyProjectDataToPrm has not started jet, or is running" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
+			echo -e "copyProjectDataToPrm has not started yet, or is running" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
 			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "copyProjectDataToPrm has not started yet or is running for project ${project}"
 			continue
 		else
 			echo -e "copyProjectDataToPrm.started file is OLDER than 4 hours.\n" \
 			"time ${run}.copyProjectDataToPrm.started was last modified:\n" \
-			"$(stat -c %y "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrm.started")" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
+			ssh "${HOSTNAME_PRM}" "stat -c %y "${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrm.started"" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
```
Comment (pneerincx) [inline bin/pipelineTiming.sh:328]:
```suggestion
			ssh "${HOSTNAME_PRM}" "stat -c %y \"${TMP_ROOT_DIR}/logs/${project}/${run}.copyProjectDataToPrm.started\"" >> "${TMP_ROOT_DIR}/logs/${project}/${run}.${SCRIPT_NAME}.log"
```

**Review by pneerincx — DISMISSED** [review]
See inline suggestions.

**Review by kdelange — APPROVED** [review]

**Pull request merged**